clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/spark/SparkConstants.java

```diff
@@ -0,0 +1,48 @@
+package com.linkedin.venice.hadoop.spark;
+
+import static org.apache.spark.sql.types.DataTypes.BinaryType;
+import static org.apache.spark.sql.types.DataTypes.IntegerType;
+
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+
+
+public class SparkConstants {
+  // Required column names for input dataframes
+  public static final String KEY_COLUMN_NAME = "key";
+  public static final String VALUE_COLUMN_NAME = "value";
+
```

Comment:
That's how I had it initially, but I think it is better to keep them as different constants (even though they have the same values) as it will prevent accidental changes that would break the contract of the input formats

Reply:
Can you explain how this can happen? I mean under what circumstances we will change `DEFAULT_KEY_FIELD_PROP` and `DEFAULT_VALUE_FIELD_PROP`?